### PR TITLE
Do a shallow clone for Docker tests.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Just saves a bit of time and work since we don't need the full history.

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
I just realized we always pull right after - but I don't know why we do that either. Do we need that step @weswigham?

**Comment by weswigham** [comment]
Probably just force of habit on my part; I always pull after a clone to fetch remote refs, and that may have made its way into the scripts.

**Comment by DanielRosenwasser** [comment]
Okay, cool! Newest change doesn't pull either.

@typescript-bot test this

**Comment by weswigham** [comment]
@typescript-bot user test this

**Comment by DanielRosenwasser** [comment]
Looks like it successfully sent a PR - all good?

**Review by weswigham — APPROVED** [review]

File: tests/cases/docker/azure-sdk/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
 FROM node:current
 RUN npm install -g @microsoft/rush
-RUN git clone https://github.com/Azure/azure-sdk-for-js.git /azure-sdk
+RUN git clone --depth 1 https://github.com/Azure/azure-sdk-for-js.git /azure-sdk
```
Comment (Kingwl) [inline tests/cases/docker/azure-sdk/Dockerfile:3]: We may combine these run into one line? Especially after big IO.

**Pull request merged**